src/dataDisplay/tableConstants.js

```diff
@@ -0,0 +1,10 @@
+export const DEBOUNCE_WAIT_TIME = 50;
+export const TH_MAX_WIDTH_PERCENTAGE_DESKTOP = 0.5;
+export const TH_MAX_WIDTH_PERCENTAGE_MOBILE = 0.8;
+const TH_RESIZABLE_MAX_WIDTH = Math.max(
+    TH_MAX_WIDTH_PERCENTAGE_DESKTOP,
+    TH_MAX_WIDTH_PERCENTAGE_MOBILE,
+);
+export {
+    TH_RESIZABLE_MAX_WIDTH,
+};
```

Comment:
Question: Why is this like that, versus just `export TH_RESIZABLE_MAX_WIDTH;`?

Reply:
My bad, sorry, I was planning to export all variables inside that object but forgot to do it directly, thank you.

Reply:
Np ... thanks for explaining and for fixing it! :-)
Resolving.